_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title: "Save the Date: June 24th-25th, 2024 - in-person workshop at Arizona State University"
+---
+
+<img
+src="/public/images/ASUworkshop_cactus.png"
+title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 24-25th, 2024 at Arizona State University"
+style="float: right; height: 20em; " />
+
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: right; height: 5em; " />
+
+MDAnalysis, in conjunction with the [Molecular Sciences Software Institute][]
+(MolSSI) and with the support of the [Chan-Zuckerberg Inititative][CZI] and the
+[Center for Biological Physics][CBP], is holding a **2-day** workshop on **June
+24th-25th** 2024 at **Arizona State University** in Tempe, Arizona, USA.
+
+This two-day workshop is intended to transform attendees from users to
+developers and will cover an introduction to MDAnalysis, software best
+practices, and guide participants through the creation of their own [MDAKit][mdakits].
+
+### Workshop outline
+
+MDAnalysis developers and experienced MolSSI instructors will teach
+participants how to build their own MDAnalysis-based Python packages by guiding
+them through the development of their own MDAKits. Specifically, the workshop
+will include 3 modules: (1) an introduction to using and writing custom
+analyses in MDAnalysis; (2) an overview of software development and maintenance
+best practices; and (3) an interactive hackathon session where participants
+write their own MDAKits.
+
+The program will run from the morning of Monday, June 24th until the early
+evening of Tuesday June 25th. 
+
+Parts of the workshop will be held in a hybrid mode for those unable to attend
+in-person. Registration for the hybrid components is required. Recordings of
+hybrid sessions will be made available after the workshop.
+
+Breakfast and lunch will will be provided together with tea/coffee during
+breaks.
+
+
+
+### Registration
+
+Attendance at this workshop will be *free* but limited to 40 in-person
+participants.
+
+A number of **bursaries** will be available to assist with travel and
+accommodation costs for participants from outside the Phoenix area. Bursaries
+(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
+accommodation* (single room within 5 minutes of the workshop venue) will be
+awarded.
+
```

Comment:
Do we want to make a single registration button here?

Reply:
I added one that looks like this

![image](https://github.com/MDAnalysis/MDAnalysis.github.io/assets/237980/dda4e7fa-1a8f-4bd7-b22a-b3d7932a3817)